src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,91 @@
+package duke.storage;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Implements file storage.
+ *
+ * @author Audrey Felicio Anwar
+ */
+public class Storage {
+    private final File data;
+
+    /**
+     * Intializes a Storage object.
+     *
+     * @param filePath The location of the storage.
+     * @throws DukeException If directory not found.
+     */
+    public Storage(String filePath) throws DukeException {
+        try {
+            File data = new File(filePath);
+            if (!data.exists()) {
+                data.createNewFile();
+                assert data.exists() : " Directory cannot be made";

```

Comment:
I am not sure what this assertion is trying to do and what is the meaning of the message " Directory cannot be made". Could you please clarify?